source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
                         <specDesc key="treatSched"/>
                      </specList>
                   </p>
+                  <p>
+                     <specList>
+                        <specDesc key="acquisition"/>
+                        <specDesc key="provenance"/>
+                     </specList>
+                  </p>
+                  <p><gi scheme="MEI">acquisition</gi> is a container for recording the process of the acquisition of an item by the holding institution. In comparison, <gi scheme="MEI">provenance</gi> deals with the history of ownership or custodianship of an item. Both elements allow for the choice of either text or more structured information when formulating the specific encoding. It is recommended to make use of <gi scheme="MEI">p</gi> elements when a text-centred encoding is favored and to use the <gi scheme="MEI">eventList</gi> element for a more structured encoding. It is up to the encoder to decide where the information is most appropriate for the particular project or encoding purposes.</p>
+                  <p>
+                     <specList>
+                        <specDesc key="exhibHist"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">exhibHist</gi> element is intended to contain descriptions of one or more public exhibitions of a bibliographic item. Often exhibitions include an additional description in the form of a tag for the public that accompanies the item on display. These descriptions may even be printed in a published exhibition catalogue, so the encoding may also include information about why the object was shown or what was significant about the exhibition. When formulating the encoding, it is at the discretion of the encoder whether to opt for text or more structured information. Text-centred encoding is made possible by <gi scheme="MEI">p</gi> elements in <gi scheme="MEI">exhibHist</gi>, among others. For more structured encodings, it is recommended to use the <gi scheme="MEI">eventList</gi> element contained in <gi scheme="MEI">exhibHist</gi>. Within MEI <gi scheme="MEI">exhibHist</gi> is conceptually bound to the item-level. As an element it is thus not permitted at the work or expression level and only permitted at the manifestation level, if the manifestation comprises a manifestation singleton.</p>
```

Comment:
`bound to the item-level` and the following is unclear here if someone consults this chapter only. So a clarification and/or reference would be good. For example:

bound to the item-level of the FRBR specification (see chapter 3.5 on FRBR above).